content/20.treatments.md

```diff
@@ -897,6 +897,24 @@ They may also engage the host immune response, encouraging the immune system to
+One type of stimulus which research indicates can induce trained immunity is bacillus Calmette-Guerin (BCG) vaccination.
+BCG is an attenuated form of bacteria _Mycobacterium bovis_.
+The vaccine is most commonly administered for the prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination (e.g., to assess if it may be effective as a potential prophylactic or potential partial prophylactic in (1) reducing susceptibility / preventing infection and (2) reducing disease severity).
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
+One set of trials, for example, uses time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many other possible factors or confounding factors (country age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, and various other factors) and (2) also it is unclear what is the relation between BCG vaccination at different ages and at longer times before the start of the SARS-CoV-2 epidemic and BCG vaccination at different ages during the SARS-CoV-2 epidemic at shorter times before the risk of possible infection; at least some of these considerations are also made or pointed out in the data analyses sited and in the article.
+The article [@doi:10.1016/j.cell.2020.04.042] also includes various related considerations such as efficacy in related animal and human studies and the safety of the vaccine, both generally and specifically with respect to SARS-CoV-2.
+Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05.008]); also, trained immunity itself may sometimes be related to either balanced, or too much or too little, or useful or harmful immunological responses.
```

Comment:
```suggestion
Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05.008]).
Trained immunity itself may sometimes also be related to either balanced, hyperactive, or hypoactive immune responses, which can correspond to useful or harmful modulation of the immunological response.
This approach should therefore be used carefully in a disease where too weak or too strong of an immune response could both be associated with increased risk.
```
Hi @yemarshall, I tried to modify this in line with your comments above, but I think the middle sentence (after the semicolon part) does need a little bit of finessing because there are so many "or"s in a row! Perhaps something like this might help? I think breaking it up into direction (balanced, accelerated, decelerated) vs effect (harmful, useful) might help make it easier to follow.

Reply:
All the sentences are related to a very important type of consideration for COVID-19/SARS-CoV-2, so splitting it up into multiple sentences risks losing the aspect that all these points are part this type of consideration. I know grammatically it can be equivalent -- but I wanted to make sure to increase the chances that the reader gets the meaning.

Reply:
Also, though earlier I noted that I liked the sound of your last added sentence suggestion -- we already made that point in general in immunology background so I preferred to not add it here and risk making a mistake.